site/app/templates/LateDaysTable.twig

```diff
@@ -1,4 +1,7 @@
+{% if (is_own_page is not defined or is_own_page == true) %}
 <div class="content">
+{% endif %}
```

Comment:
I'm really not a fan of {% if %}ing around open/close tags. We should really try to keep all tags matched so there's no chance of losing one if a condition is forgotten.